sound/soc/sof/ipc4-topology.c

```diff
@@ -1419,13 +1419,55 @@ static int sof_ipc4_control_load_volume(struct snd_sof_dev *sdev, struct snd_sof
 	return 0;
 }
 
+static int sof_ipc4_control_load_bytes(struct snd_sof_dev *sdev, struct snd_sof_control *scontrol)
+{
+	struct sof_ipc4_control_data *control_data;
+	int ret;
+
+	scontrol->size = sizeof(struct sof_ipc4_control_data) + scontrol->priv_size;
+
+	scontrol->ipc_control_data = kzalloc(scontrol->size, GFP_KERNEL);
+	if (!scontrol->ipc_control_data)
+		return -ENOMEM;
+
+	control_data = scontrol->ipc_control_data;
+	control_data->index = scontrol->index;
+	if (scontrol->priv_size > 0) {
+		memcpy(control_data->data, scontrol->priv, scontrol->priv_size);
+
+		if (control_data->data->magic != SOF_IPC4_ABI_MAGIC) {
+			dev_err(sdev->dev, "Wrong ABI magic 0x%08x.\n", control_data->data->magic);
```

Comment:
I would print more information which can help:
"%s: Wrong ABI magic (%#x) for control: %s", __func__, control_data->data->magic, scontrol->name

Reply:
As @ranj063 recommend, ```func``` is being removed from our code. I will use ```Wrong ABI magic (%#x) for control: %s\n", control_data->data->magic, scontrol->name```